tags/tags.toml

```
@@ -1181,3 +1181,11 @@ content = """
 To get multiple images to display within one embed, use multiple embeds within the same message that use the same URL `.setURL(x)`/`url: x`.
 *Note that this is meant for discord-side embeds and may break without notice.*
 """
+[invite-tracker]
+keywords = ["tracking-invites", "used-invite", "who-invited-a-user"]
+content = """ 
+Discord does not provide the invite which a member used to join. To get the invite you would have to create an invite tracking system yourself.
+These are more often than not unreliable and we do not recommend it. 
+The current recommended way is to check __manually__ by using the members tab.  
+It is unsure whether this feature will ever be available to bots
+""" 
```

- I feel like there is a lot of words not saying all that much
- I dont't think we need to address "manually check" in advice intended for bot developers (which are clearly trying to automate something here, often invite rewards or moderation features)
- added a bit of information to the unreliable part
- reworded to reflect that the "unsure if this will be available" is regarding the members tab (otherwise I already see people go "but we have the members tab now")

```suggestion

[invite-tracker]
keywords = ["tracking-invites", "used-invite", "who-invited-a-user"]
content = """ 
Discord does not provide the invite that a member used to join through the bot API.
- Tracking the uses of invites through the `inviteCreate` and `guildMemberAdd` events is unreliable and we recommend against it.
- Discord has not shared any plans to make the members tab available for bots
""" 
```

---

this indeed is better, i am not the best at properly wording stuff